pertpy/tools/_distances/_distances.py

```diff
@@ -99,6 +115,7 @@ class Distance:
     def __init__(
         self,
         metric: str = "edistance",
+        agg_fct: Literal["mean", "median", "sum"] = "mean",
```

Comment:
We could generalize this even more by making this a `Callable`. Not sure whether it's necessary, but it's something to think about.

Reply:
we use a dictionary `AGG_FCTS = {"mean": np.mean, "median": np.median, "sum": np.sum}`to convert literal to callable functions. I guess this is a way to restrict to only mean/sum/median. Or it is better to allow users to use any aggregation function they want?

Reply:
Yes, I am thinking any actually

Reply:
Thoughts @eroell ?